scene/resources/packed_scene.cpp

```diff
@@ -356,6 +356,38 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 							Array set_array = value;
 							bool is_get_valid = false;
 							Variant get_value = node->get(snames[nprops[j].name], &is_get_valid);
+
+							// For every item in the array, we'll check if it's a resource and if it needs to be duplicated.
+							// The copying is just a copy of the code above (see if (value.get_type() == Variant::OBJECT)).
+							for (int ai = 0; ai < set_array.size(); ai++)
+							{
+								Ref<Resource> res = set_array[ai];
+								if (!res.is_valid()) continue;
+								if (!res->is_local_to_scene()) continue;
+								if (n.instance >= 0) { // For the root node of a sub-scene, treat it as part of the sub-scene.
+									set_array[ai] = get_remap_resource(res, resources_local_to_sub_scene, node->get(snames[nprops[j].name]), node);
+								}
+								else {
```

Comment:
Should always be on the same line as `}`, automatic tests will complain about this

Reply:
Sorry about all that, thanks for the reviews though!

Reply:
No worries! It takes time to learn 🙂